apps/mobile-alphabet/android/app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <domain-config cleartextTrafficPermitted="true">
```

Comment:
Please research the purpose \ best-practices for this file. I am available if you want to look into it together.

Reply:
Also note that `cleartextTrafficPermitted=true` triggered a security alert on SonarCloud.

Reply:
Do some research and make sure we're not sending credentials or tokens in the clear.